content/docs/user-guide/networking/_index.md

```diff
@@ -1,30 +1,65 @@
 ---
+linkTitle: Networking
 title: Networking
+description: Learn about the low-level networking stack in Open 3D Engine (O3DE) and the Multiplayer Gem that uses it to provide multiplayer features in games and simulations.
 weight: 950
-description: The user guide for Open 3D Engine's networking stack.
 ---
 
-Open 3D Engine uses an abstracted interface layer, the `AzNetworking` framework, to provide access to OS-level networking interfaces. This layer provides access to features like network compression, encryption, and reliable UDP packets.
+**Open 3D Engine (O3DE)** provides multiple networking solutions depending on your needs:
 
-For a quick introduction to the O3DE network layer and Multiplayer Gem, watch the video below.
+* The `AzNetworking` framework, which provides a low-level networking API.
+* The **Multiplayer Gem** framework, which is built on top of `AzNetworking` and uses it to provide multiplayer networking features for games and simulations.
+* The **Remote Tools Gem**, which provides a connectivity solution between O3DE applications for debugging scenarios. Refer to the [Remote Tools Gem](/docs/user-guide/gems/reference/debug/remote-tools/) documentation for more information about this Gem.
+
+## Networking frameworks
+
+### AzNetworking
+
+`AzNetworking` is a low-level network transport interface. It provides simple, fast, and efficient networking by focusing on reduced code size and complexity, low latency on packet send and receive operations, and low message processing overhead. Features of `AzNetworking` include:
+
+* Network compression
+* Encryption using TLS/DTLS
+* Serialization
+* High-performance message processing
+* Reliable/unreliable UDP packets
+* TCP packets
+* Wrapper classes for managing UDP and TCP sockets
+
+If you need to work with networking classes at this lower level, refer to the [AzNetworking section](aznetworking/) topics and use the [AzNetworking API Reference](/docs/api/frameworks/aznetworking/annotated.html).
+
+### Multiplayer Gem
+
+The Multiplayer Gem supports entity-based asynchronous networking in O3DE using event-driven network properties and remote procedure calls to synchronize state. The Gem includes the following features that multiplayer games and other simulations commonly need:
+
+* Server authoritative networking model for anti-cheat
```

Comment:
```suggestion
* Server authoritative networking model
```
To me, anti-cheat is generally associated with specific root kitting software games integrate to prevent cheating on the client machine. I don't think further elaboration is necessary here though you could go with 'for simulation integrity'.